### PR TITLE
Ticket modal

### LINKED ISSUE #389: Ticket Modal
**Purpose**
What will this feature provide?
The ticket modal as seen in the mockups for displaying ticket details and their notes

**Proposed Solution**
If you have an idea on how to implement this feature, please share!
A new modal linked to the rows of the Tickets page and anywhere else a ticket should be shown

**Acceptance Criteria**
Please list the requirements the implementation of this feature should meet.
- [ ] Should implement the ticket modal and show the actual ticket data

**Mockup Screenshot or Page Number**
Paste a screenshot of the feature from the FIGMA mockups or cite the page number.
Pages 31 and 40:
image

**Additional context**

### PR DESCRIPTION
**Pull request by Chris-Boe**
### What issue is this solving?
Please connect this Pull Request to the issue it is resolving by using the controls on the right side of this PR.
Add the ticket modal to view ticket details
Closes issue #389  

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? No
Any special requirements to test? Go to Tickets page and click "View" on a ticket row

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code
- [x] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  SMALL: 'sm',
+  STATUS: 'status',
+  TICKET: 'ticket'
+};
+
+export const STATUS_OPTIONS = {
+  NEW: 'New',
+  IN_PROGRESS: 'In Progress',
+  RESOLVED: 'Resolved',
+  REOPEN: 'Reopen'
```
Comment (NickSchimek) [inline src/constants/index.js]: Is this constant defining the different states a ticket can be in?

If so, I'm under the impression there should only be three states. `New`, `In Progress`, and `Closed`. This is shown on page 21 of the Figma doc.

Comment (Chris-Boe) [reply src/constants/index.js]: Thanks! These have been updated

**Review by htharker42 — DISMISSED** [review]
Great work, looks awesome!

**Comment by Chris-Boe** [comment]
Updated the Tickets table with a couple features:
* Status column for ease and sorting
* ID column is now the Tenant Name + Ticket Issue text and is clickable to open the ticket 
image

**Review by htharker42 — APPROVED** [review]

**Pull request merged**